safe/gui/tools/multi_exposure_dialog.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+
+"""Multi Exposure Tool."""
+
+import logging
+
+
+from PyQt4.QtGui import QDialog, QComboBox, QLabel
+from safe.definitions.exposure import exposure_all
+from safe.utilities.resources import (
+    get_ui_class,
+)
+from safe.utilities.i18n import tr
+
+LOGGER = logging.getLogger('InaSAFE')
+FORM_CLASS = get_ui_class('multi_exposure_dialog_base.ui')
+
+
+class MultiExposureDialog(QDialog, FORM_CLASS):
+
+    """Dialog for multi exposure tool."""
+
+    def __init__(self, parent=None):
+        """Constructor for the multi exposure dialog.
+
+        :param parent: Parent widget of this dialog.
+        :type parent: QWidget
+        """
+        QDialog.__init__(self, parent)
+        self.parent = parent
+        self.setupUi(self)
+        self._create_exposure_combos()
+
+    def _create_exposure_combos(self):
+        """Create one combobox for each exposure and insert them in the UI."""
+        for one_exposure in exposure_all:
+            label = QLabel(one_exposure['name'])
+            combo = QComboBox()
+            self.frame_exposures.addWidget(label)
+            self.frame_exposures.addWidget(combo)
+
+
```

Comment:
__[flake8]__
* [W391] blank line at end of file

<!-- Comment by [SideCI](https://sideci.com) -->